### PR TITLE
Color Picker: Fix broken UI

### LINKED ISSUE #74233: Color Picker: UI is broken
We recently noticed that on latest main that the color picker UI seems to be broken and not overlaying with the correct dimensions. The underlying functionality does not seem to be impacted, so this is more of a graphical glitch.

Investigate this and figure out what changes caused this regression. Implement an appropriate fix.

It seems [this PR](https://github.com/grafana/grafana/pull/71374) most likely caused the regression.

Image

https://github.com/grafana/grafana/assets/22381771/3edac0bb-0d3d-4c9f-9e48-b2edcc8ca73f

### PR DESCRIPTION
**Pull request by nmarrs**
**What is this feature?**

Try and fix regression in the color picker. The `gridColumnGap` seems to be causing issues, though when removed the result is not 1-1 to what the color picker was back in v10.0.x

v10.0.x
Screenshot 2023-08-31 at 7 44 46 PM

with this WIP PR changes (notice color labels are closer to color options)
Screenshot 2023-08-31 at 8 08 29 PM

current main
Screenshot 2023-08-31 at 11 32 52 AM

**Which issue(s) does this PR fix?**:

Fixes #74233

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: packages/grafana-ui/src/components/ColorPicker/ColorPicker.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const colorPickerFactory = <T extends ColorPickerProps>(
         onChange,
       });
       return (
-        <PopoverController content={popoverElement} hideAfter={300}>
+        <PopoverController content={popoverElement} hideAfter={100000}>
```
Comment (nmarrs) [inline packages/grafana-ui/src/components/ColorPicker/ColorPicker.tsx:45]: This change is temporary to aid with local debugging via chrome dev tools

**Comment by torkelo** [comment]
@nmarrs do you know what change triggered the bug?

**Comment by eledobleefe** [comment]
Hi,
I took a screenshot of the `ColorPicker` in v9.1.0 and, as you can see, the labels of `Transparent` and `Text color` are not there also.
Captura de pantalla 2023-09-01 a las 10 57 57

**Comment by eledobleefe** [comment]
Hi @nmarrs! I used `git bisect` and this seems to be triggered by [this PR](https://github.com/grafana/grafana/pull/72666), more specifically [this line](https://github.com/grafana/grafana/blob/a74b088025a3fcf35a9c2e676d4c18be06e15e39/packages/grafana-ui/src/components/ColorPicker/ColorSwatch.tsx#L31). I've checked on main and going back to `!!` it seems fixed.

**Comment by nmarrs** [comment]
Fixed by https://github.com/grafana/grafana/pull/74280 (thank you @eledobleefe 🙏)

**Pull request closed without merging**